sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/policy/RequestRetryOptions.java

```diff
@@ -152,7 +144,7 @@ long calculateDelayInMs(int tryCount) {
         long delay;
         switch (this.retryPolicyType) {
             case EXPONENTIAL:
-                delay = (pow(2L, tryCount - 1) - 1L) * this.retryDelayInMs;
+                delay = ((tryCount - 1) * (tryCount - 1) - 1L) * this.retryDelayInMs;
```

Comment:
These two returns two different results.

Original one is : 2 ^ (tryCount-1)
Now: (tryCount -1) ^ 2

What if tryCount = 1, does delay expect to be negative?

Reply:
Fixing in #5469